dts/bindings/sensor/ti,tmp1075.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 Arrow Electronics.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  TMP1075 Digital Temperature Sensor.  See more info at
+  https://www.ti.com/product/TMP1075
+
+compatible: "ti,tmp1075"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
```

Comment:
incosistent "." placement in sentences, sometimes you do, sometimes you dont

Reply:
added periods.